bsonutil/indexes_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) MongoDB, Inc. 2014-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package bsonutil
+
+import (
+	"github.com/mongodb/mongo-tools-common/testtype"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func TestIsIndexKeysEqual(t *testing.T) {
+	testtype.SkipUnlessTestType(t, testtype.UnitTestType)
+
+	tests := []struct {
```

Comment:
One more suggested test case: different length index keys. For example, `{a: 1} != {a: 1, b: 1}`

Reply:
Cool! Adding the test.